framework-plugins/lisk-framework-http-api-plugin/test/functional/transactions.spec.ts

```diff
@@ -239,11 +239,7 @@ describe('Hello endpoint', () => {
 				// Assert
 				expect(status).toEqual(409);
 				expect(response).toEqual({
-					errors: [
-						{
-							message: `Failed to validate signature ${newSignature.toString('hex')}`,
-						},
-					],
+					error: `Failed to validate signature ${newSignature.toString('hex')}`,
```

Comment:
not related to this PR, but why `${newSignature.toString('hex')}` is `hex`?

Reply:
Not sure; why the original author used hex?

Reply:
As discovered hexToBuffer is being use will fix and send in this PR or separated PR based on number of test fixes required

Reply:
Fixed and updated some unit tests and functional http.